examples/commoncrawl/gradle-dhs.properties

```diff
@@ -0,0 +1,120 @@
+# These settings are used by Data Hub when
```

Comment:
We don't need this file - if someone wants to connect to a DHS instance, they'll pass in hubDhs=true and hubSsl=true .